docs/host/host.md

```diff
@@ -251,3 +251,32 @@ When Ksmtuned starts, initialize `pages_to_scan` in KSM to 1000 (**Minimum Pages
 KSM starts when the available memory falls below the **Threshold Coefficient**. If it detects that it is running, `pages_to_scan` increments by 300 (**Boost**) every minute until it reaches 5000 (**Maximum Pages**).
 
 KSM will stop when the available memory is above the **Threshold Coefficient**. If it detects that it is stopped, `pages_to_scan` decrements by 100 (**Decay**) every minute until it reaches 1000 (**Minimum Pages**).
+
+## NTP Configuration
+
+Time synchronization is important for the distributed cluster architecture, so there is a simple way to configure the NTP servers on Harvester.
+
+In the previous version, Harvester only supported NTP configuration [during the installation process](https://docs.harvesterhci.io/v1.2/install/harvester-configuration#osntp_servers). If you want to change the NTP configuration, it was necessary to manually update the configuration file on each node.
+
+In v1.2.0, Harvester supports NTP configuration in Harvester/Settings. You can configure NTP servers any time for the whole Harvester cluster. The NTP configuration will be applied to all nodes in the cluster.
```

Comment:
```suggestion
Beginning with version v1.2.0, Harvester is supporting NTP configuration on the Harvester UI Settings screen (**Advanced** > **Settings**). You can configure NTP settings for the entire Harvester cluster at any time, and the settings are applied to all nodes in the cluster.
```

Reply:
The phrase "configure NTP servers" implies that the user is configuring the actual server, which is not the case here. "NTP settings" covers server addresses and other settings related to the specified servers.